src/js/midi/midi.js

```diff
@@ -148,7 +146,7 @@ class MIDI extends EventEmitter {
               case commands.pitchbend:
                 {
                   const [low, high] = params
-                  this.emit('pitchbend', (((high << 8) | low) / 0x3fff - 1) * 100)
+                  this.emit('pitchbend', (((high << 7) | low) / 0x3fff - 1) * 100)
```

Comment:
What units is this supposed to be in? Please check that the result for `high = 0, low = 0` is of opposite sign of `high = 127, low = 127` and that `high = 64, low = 0` gives a zero.

Reply:
This should reduce the value to be between -100 and +100, but I'll write some tests to see if it works.

Reply:
Did you write the test or is this going to be replaced with WebMidi.js anyway so it doesn't matter?

Reply:
Exactly, if there are plans for larger rewrites, then I would not spend time debugging this code, but only stabilizing it.